credential/store/src/main/java/org/wildfly/security/credential/store/_private/ElytronMessages.java

```diff
@@ -140,4 +140,8 @@ CredentialStoreException invalidCredentialStoreEntryType(Class<? extends KeyStor
     @Message(id = 12000, value = "The credential store file %s does not exist or cannot be accessed.")
     CredentialStoreException credentialStoreFileDoesNotExist(String fileLocation);
 
+    @Message(id = 12001, value = "No value was found for %s in the key store")
+    @LogMessage(level = WARN)
+    void logFailedToReadValueFromKeyStore(String attribute, @Cause Throwable cause);
```

Comment:
Is this one used anywhere? It's possible I missed it.

Reply:
Sorry, forgot to remove this. Fixed, thanks!